app/services/attribute_asserter.rb

```diff
@@ -96,7 +96,8 @@ def authn_request_bundle
   end
 
   def ial2_authn_context?
-    Saml::Idp::Constants::IAL2_AUTHN_CONTEXTS.include? authn_context
+    (Saml::Idp::Constants::IAL2_AUTHN_CONTEXTS.include? authn_context) ||
```

Comment:
nit: I think this would be clearer with the paren after `.include?` ex:

```ruby
Saml::Idp::Constants::IAL2_AUTHN_CONTEXTS.include?(authn_context) ||
 # ...
```